src/en/config-azure.md

```diff
@@ -1,123 +1,194 @@
-# Configuring for Windows Azure
+Title: Juju Azure provider  
 
-This process requires you to have a Windows Azure account. If you have not
-signed up for one yet, it can obtained at http://azure.microsoft.com/en-us/.
 
-You should start by generating a generic configuration file for Juju, using the
-command:
+# Configuring for Azure 
+
+The default behaviour is for units of a service to be allocated to
+machines in a service-specific Availability Set. Read the
+[Azure SLA](https://azure.microsoft.com/en-gb/support/legal/sla/) to learn how
+availability sets affect uptime guarantees.
+
+!!! Warning: Azure has two concurrent UI versions: the older "classic" console
+(https://manage.windowsazure.com) and the "new" console
+(https://portal.azure.com). The features necessary in this document are only
+available in the classic portal.
+
+
+## Prerequisites and installation of Juju
```

Comment:
Pre-requisites yes, no to the install of Juju. People are coming here largely from the getting started page, where they have installed juju